File: Exercises/GettersSetters-exercise/GettersSetters/ViewController.m
//
//  ViewController.m
//  GettersSetters
//
//  Created by Keli'i Martin on 2/25/16.
//  Copyright © 2016 WERUreo. All rights reserved.
//

#import "ViewController.h"
#import "Vehicle.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.

    Vehicle *car = [[Vehicle alloc] init];
    car.odometer = -100;

    NSLog(@"Odometer: %lu", car.odometer);

    car.model = @"Honda Civic";

    NSLog(@"I drive a %@", car.model);
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Exercises/Animations-exercise/Animations/ViewController.swift
//
//  ViewController.swift
//  Animations
//
//  Created by Keli'i Martin on 1/27/16.
//  Copyright © 2016 WERUreo. All rights reserved.
//

import UIKit

class ViewController: UIViewController
{

    @IBOutlet var alienImage: UIImageView!

    override func viewDidLoad()
    {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning()
    {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    /*
    override func viewDidLayoutSubviews()
    {
        alienImage.frame = CGRectMake(100, 20, 0, 0)
    }

    override func viewDidAppear(animated: Bool)
    {
        UIView.animateWithDuration(1) { () -> Void in
            self.alienImage.frame = CGRectMake(100, 20, 100, 200)
        }
    }

*/
    @IBAction func updateImage(sender: AnyObject)
    {
        alienImage.image = UIImage(named: "frame5.png")
        print("Button clicked")
    }
}

